styles.css

```diff
@@ -0,0 +1,53 @@
+*{
+  font-family: Arial, Helvetica, sans-serif;
+  background-color: rgb(233, 224, 224);
+}
+
+#main{
+  text-align: center;
+}
+
+#title{
+ 
+  
+}
```

Comment:
- It would be good not to have an empty ID like this without good reason as it is best practice.

Reply:
- [ ] **Good use of use the [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) 👍 👍**
   However, It would be better to make this comment clear and easy to understand. 👍 to help the writer and also if possible you may try to give an example to make it easier 😎😃

- [ ] **[Optional]** I think It would be better to use a couple of emojis or gifs in the review to make the code review more human-friendly 👽😍